resources/migrations/001_update_migrations.yaml

```diff
@@ -7042,6 +7088,7 @@ databaseChangeLog:
       id: v50.2024-02-26T22:15:55
       author: noahmoss
       comment: New `create_queries` permission
+      validCheckSum: 9:7c2bc517b6def4e3278394b0399c3d4b
```

Comment:
I'm modifying this migration in-place, so I've set the old checksum as a valid checksum so that it doesn't error for folks who have already run it.